src/main/java/edu/harvard/iq/dataverse/DOIEZIdServiceBean.java

```diff
@@ -29,7 +30,8 @@
     private String USERNAME;
     private String PASSWORD;
 
-    public DOIEZIdServiceBean() {
+    @PostConstruct
+    private void loginToEZId() {
         logger.log(Level.FINE,"Constructor");
```

Comment:
Change logging message to specify it's called from "PostConstruct" and not "Construct"